packages/augur-core/tests/old_eth_utils.py

```diff
@@ -0,0 +1,546 @@
+
```

Comment:
This file's name makes me think it's deprecated. Is there a better name? Or add a file-level docstring.

Reply:
Not really deprecated in this instance since it'll probably stay here forever. Just a port of the old pyethereum utils since they're heavily depended on in one of the test suites